db/migrate/20220415125558_create_members.rb

```diff
@@ -0,0 +1,15 @@
+class CreateMembers < ActiveRecord::Migration[7.0]
+  def change
+    create_enum :member_role, %w[participant judge]
+
+    create_table :members do |t|
+      t.enum :role, enum_type: :member_role, default: 'participant', null: false
+      t.references :user, index: true, foreign_key: true
+      t.references :challenge, index: true, foreign_key: true
+
+      t.timestamps
+    end
+
+    add_index :members, :role
```

Comment:
+ index user_id, challenge_id

Reply:
It looks like option `index: true` adds indices for references columns. It can be checked in [schema](https://github.com/DevChallengeIT/devchallenge-platform/pull/30/files#diff-cbf8b25d6d853acf58fa9057841f407d29ffe90649c75cf9e3f51b2d6e3aa1d3R83)